web/client/configs/new.json

```diff
@@ -1,23 +1,23 @@
 {
   "version": 2,
```

Comment:
the same applies, here, let's discuss with the team if we need to create a PR that formats the json files
please revert format changes here